@xen-orchestra/lite/src/stories/web-core/card/card-title.story.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      slot(),
+      slot('info').help('Meant to receive an info text'),
+      slot('link').help('Meant to receive a tertiary button'),
```

Comment:
Don't forget to update the story according to the changes made in the component (the `slot('link')` doesn't exist).
Also, I think it would be great to add help text even for the default slot in this case:
```suggestion
      slot().help('Meant to receive the title and a UiCounter'),
      slot('info').help('Meant to receive an info text and / or a tertiary UiButton'),
```